README.md

```diff
@@ -55,6 +55,33 @@ In ysplitter mode a third virtual port is connected and has access to the data s
 
 ![PICsimLab](ssniffer/tty0ttySerialSniffer.png "PICsimLab")  
 
+## Quickstart guide
+
+The fastest and safest way to install tty0tty dkms module or ssniffer on Debian or Ubuntu is to use the APT 
```

Comment:
I think it would be better to replace this sentence with something like: 
You can get precompiled  tty0tty dkms  module or ssniffer packages from the piduino.org repository for Debian or Ubuntu.

Reply:
it's done